DeeployTest/testUtils/testRunner.py

```
@@ -271,6 +274,8 @@ def generate_cmd_args(self) -> str:
                 command += " --randomizedMemoryScheduler"
             if self.args.profileTiling:
                 command += f" --profileTiling"
+            if self.args.profileMicrobenchmark:
+                command += f" --profileMicrobenchmark"
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)